.gitlab-ci.yml

```diff
@@ -90,13 +90,36 @@ download-single-step-artifacts:
       start_in: 15 minutes
   script:
     - .gitlab/download-single-step-artifacts.sh
+    - cp tracer/build/artifacts/requirements.json artifacts/requirements.json
   artifacts:
     expire_in: 2 weeks
     paths:
       - artifacts
 
 package-oci:
   needs: [ download-single-step-artifacts ]
+  artifacts:
+    paths:
+      - "packaging/sources/version"
+      - "packaging/sources/requirements.json"
+      - "packaging/*.tar"
+
+requirements_json_test:
```

Comment:
These changes should be ported to one-pipeline, but keeping them for here until the changes are merged to the injector